linAssgn2

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+clear
+echo "HI :) " 
+find *.log > report.txt
+find *.txt >> report.txt
+tar -zcvf mathTest.tar.gz ../linux_assgn
```

Comment:
Just for understanding the use of -zcvf tarFileName.tar.gz I have written two tar statements.
Now I have changed the code to compress only log and text files.